packages/rs-dpp/src/state_transition/state_transitions/document/documents_batch_transition/document_transition/document_create_transition/v0/mod.rs

```diff
@@ -222,6 +222,7 @@ mod test {
         let test_document_properties_alpha_identifier = Value::from([
             ("type", Value::Text("array".to_string())),
             ("byteArray", Value::Bool(true)),
+            ("position", Value::U16(0)),
```

Comment:
16 or 32?

Reply:
It won't really matter as it will get converted to a u64.

Reply:
It just will be confused for people who will read the code so they will think it's u16